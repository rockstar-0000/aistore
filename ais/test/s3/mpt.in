dd if=/dev/zero of=/tmp/testfile bs=123K count=94 // IGNORE
s3cmd --host=$HOST mb s3://$BUCKET $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)"
ais bucket props set ais://$BUCKET checksum.type=md5
s3cmd --host=$HOST put /tmp/testfile s3://$BUCKET $PARAMS --multipart-chunk-size-mb=5 --host-bucket="$HOST/%(bucket)" // IGNORE
ais ls ais://$BUCKET -H
rm /tmp/testfile // IGNORE
s3cmd --host=$HOST rb s3://$BUCKET $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)"
