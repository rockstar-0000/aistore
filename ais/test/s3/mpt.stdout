Bucket 's3://$BUCKET/' created
"checksum.type" set to:"md5" (was:"xxhash")

Bucket props successfully updated.
testfile 11.29MiB
Bucket 's3://$BUCKET/' removed
